packages/global/templates/content/index.marko

```diff
@@ -161,6 +163,7 @@ $ const displayVenueBox = ['article', 'media-gallery'].includes(type) ? true : f
         id=id
         type=type
         name=content.name
+        primary-section-alias=content.primarySection.alias
```

Comment:
@Shinsina  this should be something like `display-ads=(content.priamrySection.alias === 'sponsored')`